config/routes.d/staging.horse.json

```diff
@@ -3,7 +3,8 @@
     "routes": {
       "^/docs/private-cloud/release-notes/": "developer.rackspace.com/docs-singlepage.html",
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
-      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html"
+      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
+      "^/docs/tech-white-papers/": "support.rackspace.com/white-paper.html"
```

Comment:
The route file maps the path you configured in the content.d file to a template.  So when someone requests a page, deconst applies the specified template to the content stored at that URL and serves it in the browser.

So the left side of this key pair has to match the path you configured.

I think you want: 

``^/white-papers/``